sensu/plugins/check-mongodb.py

```diff
@@ -0,0 +1,1449 @@
+#!/usr/bin/env python
+
+#
+# A MongoDB Nagios check script
+#
+
+# Script idea taken from a Tag1 script I found and I modified it a lot
+#
+# Main Author
+#   - Mike Zupan <mike@zcentric.com>
+# Contributers
+#   - Frank Brandewiede <brande@travel-iq.com> <brande@bfiw.de> <brande@novolab.de>
+#   - Sam Perman <sam@brightcove.com>
+#   - Shlomo Priymak <shlomoid@gmail.com>
+#   - @jhoff909 on github
+#   - @jbraeuer on github
+#   - Dag Stockstad <dag.stockstad@gmail.com>
+#   - @Andor on github
+#   - Steven Richards - Captainkrtek on Github <sbrichards@mit.edu>
+#
+
+# License: BSD
```

Comment:
This will add BSD licensed code to our tree, which much like our Ursula discussion will have to be vetted, approved, accounted for, etc.. I'm not keen on introducing new licenses into our tree.